AgateDecorator/AgateDecorator.js

```diff
@@ -118,7 +118,7 @@ const AgateDecorator = hoc(defaultConfig, (config, Wrapped) => {
 	if (ri) App = ResolutionDecorator(ri, App);
 	if (i18n) App = I18nDecorator(App);
 	if (spotlight) App = SpotlightRootDecorator({noAutoFocus}, App);
-	if (skin) App = Skinnable({defaultSkin: 'gallium-day'}, App);
+	if (skin) App = Skinnable({defaultSkin: 'gallium'}, App);
```

Comment:
Should this accept variants here as well?

Reply:
Perhaps but only if we want to allow overriding the supported variants. `skinVariants` is an acceptable prop on the App with `Skinnable` applied so an app can also set the variants that way iirc.